mtad.yaml

```diff
@@ -0,0 +1,55 @@
+ID: learning-management-system
+version: 1.2.0
+_schema-version: "3.1.0"
+
+modules:
+  - name: learning-management-system
+    type: java
+    path: ./target/LearningManagementSystem-1.2.0.jar
+    parameters:
+      memory: 2048M
+      buildpack: sap_java_buildpack_jakarta
+      random-route: true
+      env:
```

Comment:
`env` can be replaced with `properties`. And will be under `modules[0].properties`, not under `parameters` key